src/account/AccountStorage.sol

```diff
@@ -48,9 +48,6 @@ struct SelectorData {
 }
 
 struct AccountStorage {
-    // AccountStorageInitializable variables
-    uint8 initialized;
-    bool initializing;
```

Comment:
Could we revert this change? It goes against recommended usage of modular accounts, since it breaks cross-version upgrades if using the same OZ library. Account implementations generally want to track their state separately, and this risks the initialization state colliding.

Reply:
Hmm, I think upgradeability of an account is not a high priority for a ref impl, SimpleAccount has  the same pattern. Doing it this way cuts a lot of LOC out. Agree that we want to make sure that people are aware about this though, is there a better way to document this?

Reply:
I agree on simplicity, but I think the reference implementation should avoid having significant security caveats / limitations that need to be called out separately, because it distracts from the core understanding of the system. This is a common enough workflow that most account implementations should be following, so I'd prefer to keep it in.

Reply:
Since we're already using namespaced storage for other state, I'm leaning towards keeping these in namespaced storage as well.

Reply:
Reverted changes, but OZ deprecated Address.isContract in OZ 5.0

Since 5.0 initializiable is also more readable, I copied the logic over